### PR TITLE
(doc) Fix typos and spelling errors

### PR DESCRIPTION
**Pull request by sinapis-amicus**
(doc)
## Description Of Changes
Fixed some minor typos and spelling errors.

## Motivation and Context
Improve code quality and documentation legibility.

### CODE REVIEW

File: src/chocolatey.sln.DotSettings [hunk 1]
```diff
@@ -4,7 +4,7 @@
...
+	<s:String x:Key="/Default/CodeStyle/CodeCleanup/Profiles/=sensible/@EntryIndexedValue">&lt;?xml version="1.0" encoding="utf-16"?&gt;&lt;Profile name="sensible"&gt;&lt;CSArrangeThisQualifier&gt;True&lt;/CSArrangeThisQualifier&gt;&lt;CSMakeFieldReadonly&gt;True&lt;/CSMakeFieldReadonly&gt;&lt;CSOptimizeUsings&gt;&lt;OptimizeUsings&gt;True&lt;/OptimizeUsings&gt;&lt;EmbraceInRegion&gt;False&lt;/EmbraceInRegion&gt;&lt;RegionName&gt;&lt;/RegionName&gt;&lt;/CSOptimizeUsings&gt;&lt;CSShortenReferences&gt;True&lt;/CSShortenReferences&gt;&lt;CSReformatCode&gt;True&lt;/CSReformatCode&gt;&lt;CSRemoveCodeRedundancies&gt;True&lt;/CSRemoveCodeRedundancies&gt;&lt;CSUseAutoProperty&gt;True&lt;/CSUseAutoProperty&gt;&lt;CSUseVar&gt;&lt;BehaviourStyle&gt;CAN_CHANGE_TO_IMPLICIT&lt;/BehaviourStyle&gt;&lt;LocalVariableStyle&gt;IMPLICIT_WHEN_INITIALIZER_HAS_TYPE&lt;/LocalVariableStyle&gt;&lt;ForeachVariableStyle&gt;ALWAYS_EXPLICIT&lt;/ForeachVariableStyle&gt;&lt;/CSUseVar&gt;&lt;CSharpFormatDocComments&gt;True&lt;/CSharpFormatDocComments&gt;&lt;CSUpdateFileHeader&gt;True&lt;/CSUpdateFileHeader&gt;&lt;/Profile&gt;</s:String>
```
Comment (gep13) [inline src/chocolatey.sln.DotSettings:7]: While I understand that this is a spelling mistake, this is also not a file that we update manually, this is controlled via the ReSharper plugin for Visual Studio.  As a result, I am hesitant to make this change, given that we don't know whether ReSharper works with the modified spelling, or expects the misspelling.

**Comment by gep13** [comment]
@sinapis-amicus thank you very much for taking the time to find, and also to correct, these spelling mistakes!

I have left one comment regarding one of the changes you have suggested, can you let me know what you think?

Also, can you update the commit message for the commit to include `(doc) ` at the start of the line?  This would be in-keeping with our standard commit message formatting.

File: src/chocolatey.sln.DotSettings (hunk 1 above)
Comment (gep13) [reply src/chocolatey.sln.DotSettings]: I have taken the liberty to update this PR locally on my machine, and will push the changes back to this PR.

**Comment by gep13** [comment]
@sinapis-amicus actually, I won't be able to push the changes back to this PR branch, since you haven't created the PR from a branch on your fork, but rather directly from your develop branch.  The [contributing document](https://github.com/chocolatey/choco/blob/develop/CONTRIBUTING.md#contributing-process) explains the process that we like to see folks following when suggesting changes to this repository.

Since I have already made them changes locally, including rebasing the 3 commits here into a single PR, I will go ahead and merge that directly, and close this PR.  Your contributions will still be correctly attributed to you, it is just that the commits won't be coming from this PR.

Thanks again for your help in fixing these spelling mistakes, appreciate it!

**Comment by gep13** [comment]
This PR has been rebased and merged into the develop branch here: https://github.com/chocolatey/choco/commit/f2b05909ee1859a5341ff628c4a6fb7a60798dc1

**Comment by sinapis-amicus** [comment]
> @sinapis-amicus actually, I won't be able to push the changes back to this PR branch, since you haven't created the PR from a branch on your fork, but rather directly from your develop branch. The [contributing document](https://github.com/chocolatey/choco/blob/develop/CONTRIBUTING.md#contributing-process) explains the process that we like to see folks following when suggesting changes to this repository.
> 
> Since I have already made them changes locally, including rebasing the 3 commits here into a single PR, I will go ahead and merge that directly, and close this PR. Your contributions will still be correctly attributed to you, it is just that the commits won't be coming from this PR.
> 
> Thanks again for your help in fixing these spelling mistakes, appreciate it!

Thanks Gary.

Happy New Year

**Comment by gep13** [comment]
> @sinapis-amicus said...
> Thanks Gary.
> 
> Happy New Year

You are very welcome! And to you!

**Pull request closed without merging**